src/Libraries/RevitNodes/Elements/RoofType.cs

```diff
@@ -1,31 +1,29 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Linq;
+using Autodesk.DesignScript.Runtime;
+using Autodesk.Revit.DB;
 using RevitServices.Persistence;
 
 namespace Revit.Elements
 {
     /// <summary>
     /// A Revit RoofType
     /// </summary>
-    public class RoofType : Element
+    public class RoofType : ElementType
```

Comment:
for any of these types which now have new base classes, it might be fine, or it might not, it's worth considering a package which used this type as the input to a node or even just internally, will this code need to be recompiled? Will it need to have changes made and then recompiled? 

Can you prove it won't?

Reply:
also, it looks like you did not have to modify any existing tests? That is usually a good sign for these type of refactors.

I think the best that can be done, if you want to move forward with a refactor like this is to try this change out on a package that was compiled against an old version of DynamoRevit and see what happens. This can be run as an automated test.

also, please see the one change which looks to be a genuine break.